src/dxtbx/format/FormatCBFMultiTile.py

```diff
@@ -127,17 +132,18 @@ def _detector(self):
             size = tuple(reversed(cbf.get_image_size(0)))
 
             try:
-                cbf.find_category(b"array_intensities")
-                cbf.find_column(b"undefined_value")
-                cbf.select_row(i)
-                # undefined_value, interpreted as 1 less than the minimum acceptable value
-                underload = cbf.get_doublevalue()
-                overload = cbf.get_overload(i)
-                trusted_range = (underload + 1, overload)
-            except Exception as e:
-                if "CBF_NOTFOUND" not in str(e):
-                    raise
-                trusted_range = (0.0, 0.0)
+                underload = find_underload_value(cbf)
+            except Exception:
+                try:
+                    underload = find_undefined_value(cbf) + 1
+                except Exception:
+                    underload = 0
+            try:
+                overload = cbf.get_overload(i) - 1
+            except Exception:
+                overload = 1.0e6
+
+            trusted_range = (underload, overload)
```

Comment:
I really hate the terms "underload" and "overload". I recognise they are part of CBFlib, but

> The valid pixel values are those less than
              [_array_intensities.overload](http://www.bernstein-plus-sons.com/software/CBF/doc/cif_img_1.8.4.html#_array_intensities.overload) and greater than or equal to 
              [_array_intensities.underload](http://www.bernstein-plus-sons.com/software/CBF/doc/cif_img_1.8.4.html#_array_intensities.underload)

makes it sound like the `trusted_range` for a detector with int type images should be `(underload, overload - 1)`.

This confusion is why in #536 I preferred terms like `min_trusted_value` and `max_trusted_value`.

Reply:
Comment added, and after more thought on it, I think your way is good, so updated variable names, including those back in detector.py for cbf full.